Sources/Plasma/PubUtilLib/plDrawable/plDynaDecalMgr.cpp

```diff
@@ -210,8 +210,8 @@ void plDynaDecalMgr::Read(hsStream* stream, hsResMgr* mgr)
         mgr->ReadKeyNotifyMe(stream, new plGenRefMsg(GetKey(), plRefMsg::kOnCreate, 0, kRefPartyObject), plRefFlags::kPassiveRef);
     }
 
-    fMaxNumVerts = (uint16_t)(stream->ReadLE32());
-    fMaxNumIdx = (uint16_t)(stream->ReadLE32());
+    fMaxNumVerts = stream->ReadLE32();
+    fMaxNumIdx = stream->ReadLE32();
```

Comment:
FWIW, vertex and index buffers are limited to 16-bits. Not that I'm suggesting changing anything here, just making a note.